subseas_flex_fcst/config-zmd.yaml

```diff
@@ -0,0 +1,24 @@
+# App set up
+server: 0.0.0.0
+port: 8063
+
+# Forecast options
+results_path: /data/remic/mydatafiles/ZMD/nextgenzmd
```

Comment:
It's confusing to call this "results" when it's the input to the application. `forecast_path`?